PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -19,12 +19,13 @@ public static void Main(string[] args)
                     numberCorrect++;
                 }
             }
+
             Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
         }
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (double) numberCorrectAnswers / numberOfQuestions * 100 + "%";
```

Comment:
Only thing I may have done differently would have been to apply rounding in case the percentage did not come back as a whole number.
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```